README.markdown

```diff
@@ -3138,6 +3139,34 @@ See also [lua_ssl_trusted_certificate](#lua_ssl_trusted_certificate).
 
 [Back to TOC](#directives)
 
+lua_ssl_conf_command
+--------------------
+
+**syntax:** *lua_ssl_conf_command &lt;command&gt;*
+
+**default:** *no*
+
+**context:** *http, server, location*
+
+Sets arbitrary OpenSSL configuration [commands](https://www.openssl.org/docs/man1.1.1/man3/SSL_CONF_cmd.html).
+
+The directive is supported when using OpenSSL 1.0.2 or higher and nginx 1.19.4 or higher. According to the specify command, higher OpenSSL version may be needed.
+
+Several `lua_ssl_conf_command` directives can be specified on the same level:
+
+```
+ssl_conf_command Options PrioritizeChaCha;
+ssl_conf_command Ciphersuites TLS_CHACHA20_POLY1305_SHA256;
+```
+
+Configuration commands are applied after lua-nginx own configuration for SSL, so they can be used to override anything set by lua-nginx. 
+
+Note though that configuring OpenSSL directly with `lua_ssl_conf_command` might result in a behaviour nginx does not expect, and should be done with care.
+
+This directive was first introduced in the `v0.10.20` release.
```

Comment:
Should be `v0.10.21`?

Reply:
I‘m not sure about it. I noticed the current version is `v0.10.20rc1`, so I thought the next release version would be `v0.10.20`..

Reply:
I‘m not sure about it. I noticed the current version is `v0.10.20rc1`, so I thought the next release version would be `v0.10.20`..